Middleware/src/main/java/ca/concordia/ngci/tools4cities/middleware/consumers/SortedEnergyConsumptionConsumer.java

```diff
@@ -0,0 +1,83 @@
+package ca.concordia.ngci.tools4cities.middleware.consumers;
+
+import java.util.*;
+
+import ca.concordia.ngci.tools4cities.middleware.middleware.AbstractConsumer;
+import ca.concordia.ngci.tools4cities.middleware.middleware.IConsumer;
+import ca.concordia.ngci.tools4cities.middleware.middleware.IProducer;
+
+public class SortedEnergyConsumptionConsumer extends AbstractConsumer<String> implements IConsumer<String> {
```

Comment:
Please add a comment block to describe the purpose of the class, similarly to what you did in previous PRs